Examples/Scripts/KF_timing.py

```diff
@@ -0,0 +1,55 @@
+import ROOT
```

Comment:
I think we will have to do a cleanup of the scripts, probably as preparation for the Tutorial/Workshop.
This is unrelated to this PR though, it should go into an issue.

Reply:
That's a good idea!